.github/workflows/cleanup-images.yml

```diff
@@ -0,0 +1,23 @@
+name: Delete old container images
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 6 1 * *"  # every day at 6 in the morning
+
+jobs:
+  clean-ghcr:
+    name: Delete old unused container images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete untagged and dev containers older than a week
+        uses: snok/container-retention-policy@v1.5.1
+        with:
+          image-names: epinio-server, epinio-unpacker
+          cut-off: A week ago UTC
+          account-type: org
+          org-name: epinio
+          keep-at-least: 1
+          filter-tags: v*-g*
```

Comment:
Maybe it's not needed but I would check also for the commit number, if any: `v*-[0-9].-g*`

`v1.2.0-78-g535d6393`

I don't think we are going to have an architecture that will start with `g` (like `v1.2.0-g64`) but... who knows.

Reply:
How many digits does the commit number have ? IIRC it is varying, i.e 1/2/... digits, not fixed.
As the pattern is glob-style, not regexp, it would have to be `v*-[0-9]*-g*`, right ? (No `.`)